introduction/architecture.md

```diff
@@ -111,6 +110,68 @@ while maintaining support for multiple indexes.
 For more on the motivation and design of TimescaleDB's adaptive space/time
 chunking, please see our [technical blog post][chunking].
 
+## TimescaleDB Clustering (PRIVATE BETA) [](timescaledb-clustering)
+
+TimescaleDB supports multi-node clustering by leveraging the hypertable and chunk primitives described above.
+In the multi-node topology, all nodes are TimescaleDB instances, but may play different roles
+depending on their responsibilities within a distributed database.
+
+### Node Types
+
+The two roles that nodes can take on are:
+1) Access node
+2) Data node
+
+A front-end access node accepts all requests from the client, distributes the requests and queries
+appropriately to back-end data nodes, and aggregates the results received from the data nodes.
+Access nodes store cluster-wide information about the different data nodes as well as how
+chunks are distributed across those data nodes. Access nodes can also store non-distributed
+hypertables, as well as regular PostgreSQL tables.
+
+Back-end data nodes store data associated with distributed hypertables. Data nodes
+are not aware of cluster-wide distribution policies, and otherwise look just as if they were
+single TimescaleDB instances. However, certain actions are blocked on data nodes to protect
```

Comment:
What actions are blocked?

Reply:
I *believe* we block any sort of inserts / modifications? @WireBaron do you remember?

Reply:
Not yet. There is an issue in Pivotal to block inserts: https://www.pivotaltracker.com/story/show/166813477

Reply:
Can you consider to change formulation from being blocked to:

> Avoid direct access to local hypertable or chunks of the distributed hypertable as it can fail or make the distributed hypertable inconsistent.

Reply:
> Can you consider to change formulation from being blocked to:
> 
> > Avoid direct access to local hypertable or chunks of the distributed hypertable as it can fail or make the distributed hypertable inconsistent.

Actually it is already partly addressed in the next sentence.

Reply:
Done! I added in that sentence, and kept the next sentence to make it SUPER clear not to touch the data node.